src/FormCheck.tsx

```diff
@@ -65,7 +63,11 @@ const propTypes = {
    */
   as: PropTypes.elementType,
 
-  /** A HTML id attribute, necessary for proper form accessibility. */
+  /**
+   * A HTML id attribute, necessary for proper form accessibility.
+   *
+   * This is **required** when `type="switch"`.
+   */
```

Comment:
The ID also ensures that clicking on the label works for _any_ check, in the Switch case, the whole component is built around that behavior

Reply:
I thought that might be implied with the accessibility bit.  Did you want to make this more explicit in the docs?

Reply:
It is an a11y thing, i think it's also just a behavior that people expect and tend to forget why it works. So maybe worth mentioning explicitly just to be extra helpful?